### ISSUE TITLE
 no-missing-import does not work with "paths" from "tsconfig.json"

### ISSUE THREAD

**Issue by Zamiell** [eslint-community/eslint-plugin-n#84]
In a monorepo, you typically use the "paths" option inside of the "tsconfig.json" file to make your individual packages aware of each other like this:

#### `tsconfig.json`

```json
{
    "paths": {
      "foo": ["packages/foo/src/index.ts"],
      "bar": ["packages/bar/src/index.ts"],
    },
}
```

However, this triggers false positives for `no-missing-import`.

**Comment by wdzeng** [comment]
Any updates on this?

**Comment by scagood** [comment]
🤔 I have not had this issue in my monorepos, could you make a small reproduction repository?

---

Here are a couple of screenshots from a yarn v3 repo:

<img width="517" alt="image" src="https://github.com/eslint-community/eslint-plugin-n/assets/2230835/c0c2a17d-5d3c-440a-90b3-ee7656a8c4fb">
<img width="517" alt="image" src="https://github.com/eslint-community/eslint-plugin-n/assets/2230835/9b336cf5-e928-4507-ad6b-22cbcc240138">

---

I have not needed the "paths" property at all for importing files, I have only needed this for importing additional types that I don't directly import, eg in another package:

<img width="517" alt="image" src="https://github.com/eslint-community/eslint-plugin-n/assets/2230835/29ab1f23-7748-4d14-aef3-8658bb7ad536">

**Comment by scagood** [comment]
I think I found the issue, it turns out that when the depended upon package is not built (at least once) then the file shows up as missing!
